cmd/minikube/cmd/start.go

```diff
@@ -1386,29 +1386,26 @@ func validatePorts(ports []string) error {
 		}
 	}
 	for _, p := range exposedPorts {
-		if err := validatePort(p, false); err != nil {
+		if err := validatePort(p); err != nil {
 			return err
 		}
 	}
 	for _, p := range hostPorts {
-		if err := validatePort(p, true); err != nil {
+		if err := validatePort(p); err != nil {
 			return err
 		}
 	}
 	return nil
 }
 
-func validatePort(port string, isHost bool) error {
+func validatePort(port string) error {
 	p, err := strconv.Atoi(port)
 	if err != nil {
 		return errors.Errorf("Sorry, one of the ports provided with --ports flag is not valid: %s", port)
 	}
 	if p > 65535 || p < 1 {
 		return errors.Errorf("Sorry, one of the ports provided with --ports flag is outside range: %s", port)
 	}
-	if isHost && detect.IsMicrosoftWSL() && p < 1024 {
```

Comment:
why isHost been removed from the logic ? doesnt seem related to this PR

Reply:
The way I understand it, the current logic is "on WSL (and only on WSL), check if the host port is below 1024", and since the change is to allow those ports on WSL, we don't need any special checks for host ports.

Also if we remove this condition from function this argument is just not used anywhere anymore